kura/distrib/src/main/deb/control/postinst

```diff
@@ -3,10 +3,50 @@
 INSTALL_DIR=/opt/eclipse
 TIMESTAMP=`date +%Y%m%d%H%M%S`
 LOG=/tmp/kura_install_${TIMESTAMP}.log
+WD_TMP_FILE=/tmp/watchdog
+REFRESH_TIME=5
+
+##############################################
+# UTILITY FUNCTIONS
+##############################################
+# Start refresh watchdog device
+function startRefreshWatchdog {
+    if [ -f "${WD_TMP_FILE}" ]; then
+        WATCHDOG_DEVICE=`cat ${WD_TMP_FILE}`
+        echo "Got watchdog ${WATCHDOG_DEVICE}" >> ${LOG} 2>&1
+        refreshWatchdog &
```

Comment:
I'm paranoid but **unconditionally** refreshing the watchdog from a child process is generally not considered a good practice. The parent could die or hang (for whatever reason) and the child process will happily continue to kick the watchdog. We need to find a better way